### DISCUSSION TITLE
how to change prisma engines in Intranet

### DISCUSSION THREAD

**Discussion by SunshineHusky** [prisma/prisma#19996] [Q&A]
i'm start a  next.js project with  pirsma, my server can't connet to https://binaries.prisma.sh, when  i  excute npm i in server,

Error: request to https://binaries.prisma.sh/all_commits/b20ead4d3ab9e78ac112966e242ded703f4a052c/rhel-openssl-1.0.x/libquery_engine.so.node.sha256 failed, reason: connect ENETUNREACH 104.22.22.109:443
npm timing command:install Completed in 78470ms

how can i fix  it

**Comment by nurul3101** [comment]
Hello @SunshineHusky 👋 

Are you using any firewalls in your network?
Can you try changing your network and see if you get the same issue?

What happens when you click on this URL?https://binaries.prisma.sh/all_commits/b20ead4d3ab9e78ac112966e242ded703f4a052c/rhel-openssl-1.0.x/libquery_engine.so.node.sha256 

Does the engine file get downloaded?

**Reply by nurul3101** [reply]
@SunshineHusky I am checking to see if you had a chance to look at my last response.

It would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 
Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by nurul3101** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
